scripts/check_configs.ts

```diff
@@ -594,6 +603,26 @@ async function main() {
         }
       }
 
+      if(jsonData.market_banners) {
+        const marketBanners = jsonData.market_banners
+        const marketBannerIds = marketBanners.map((banner) => banner.market_id)
+
+        const hasInvalidMarketBannerIds = checkValidEntries(marketBannerIds, marketIds)
+        const hasDuplicateMarketBannerIds = checkDuplicateEntries(marketBannerIds)
+
+        if (hasInvalidMarketBannerIds.status && hasInvalidMarketBannerIds.entry) {
+          let listOfInvalidIds: string = hasInvalidMarketBannerIds.entry.join(", ");
+          console.error(`ERROR: ${network}.json has the following invalid market ids under the market_banners field: ${listOfInvalidIds}`)
+          outcomeMap[network] = false;
+        }
+
+        if (hasDuplicateMarketBannerIds.status && hasDuplicateMarketBannerIds.entry) {
+          let listOfDuplicates: string = hasDuplicateMarketBannerIds.entry.join(", ");
+          console.error(`ERROR: ${network}.json has duplicated market banners for the following market ids: ${listOfDuplicates}. Please make sure to input each market banner only once in ${network}`);
+          outcomeMap[network] = false;
+        }
```

Comment:
Pls move L607 - L623 into a separate function for better code organisation. You can refer to this [code snippet](https://github.com/Switcheo/demex-webapp-config/blob/feat/market-banners/scripts/check_configs.ts#L626-L628) on how it's done.